discovery-provider/integration_tests/tasks/test_index_operations.py

```diff
@@ -1,494 +1,524 @@
-#                 # Process a bunch of blocks to make sure we covered everything
-#                 task.run()
-#                 current_block_query = (
-#                     session.query(Block).filter_by(is_current=True).all()
-#                 )
-#                 current_block = (
-#                     current_block_query[0].number if len(current_block_query) > 0 else 0
-#                 )
-#         assert False
-#     except IndexingError:
-#         error = get_indexing_error(redis)
-#         assert error["message"] == "Broken session.commit"
-#         assert error["txhash"] == "commit"
-
-
-# def test_index_operations_skip_block(celery_app, celery_app_contracts, mocker):
-#     """
-#     Confirm indexer skips block when session commit fails
-#     """
-#     task = celery_app.celery.tasks["update_discovery_provider"]
-#     db = task.db
-#     web3 = celery_app_contracts["web3"]
-
-#     seed_contract_data(task, celery_app_contracts, web3)
-
-#     # patch get_tx_hash_to_skip to raise an exception
-#     class MockSkipOnlyOneBlock:
-#         skipped = False
-#         skipped_block_number = None
-
-#         def get_tx_hash_to_skip_stub(self, session, redis):
-#             if not self.skipped:
-#                 self.skipped_block_number = (
-#                     session.query(Block).filter_by(is_current=True).first().number
-#                 )
-#                 self.skipped = True
-#                 return "commit"
-#             else:
-#                 return None
-
-#     skip_block_helper = MockSkipOnlyOneBlock()
-
-#     mocker.patch(
-#         "src.tasks.index.get_tx_hash_to_skip",
-#         side_effect=skip_block_helper.get_tx_hash_to_skip_stub,
-#         autospec=True,
-#     )
-
-#     seed_contract_data(task, celery_app_contracts, web3)
-
-#     current_block = None
-#     latest_block = None
-#     with db.scoped_session() as session:
-#         # Catch up the indexer
-#         current_block_query = session.query(Block).filter_by(is_current=True).all()
-#         current_block = (
-#             current_block_query[0].number if len(current_block_query) > 0 else 0
-#         )
-#         latest_block = web3.eth.getBlock("latest", True)
-#         while current_block < latest_block.number:
-#             # Process a bunch of blocks to make sure we covered everything
-#             task.run()
-#             current_block_query = session.query(Block).filter_by(is_current=True).all()
-#             current_block = (
-#                 current_block_query[0].number if len(current_block_query) > 0 else 0
-#             )
-#         skipped_block_was_added = (
-#             len(
-#                 session.query(Block)
-#                 .filter_by(number=skip_block_helper.skipped_block_number)
-#                 .all()
-#             )
-#             == 1
-#         )
-#         assert skipped_block_was_added
+import json
+import secrets
+from contextlib import contextmanager
+
+import ipfshttpclient
```

Comment:
we can probably remove this package and the ipfs client [init](https://github.com/AudiusProject/audius-protocol/pull/2613/files#diff-7245c5e2cbc7f8f61c8275786713a48967eedbb37e06a0f301914880b638c4b4R26-R28) as well as references to ipfs

Reply:
sure - i'll do in fast-follow